constellations/orion/src/domains/user_valid_test.go

```diff
@@ -109,3 +109,70 @@ func TestValidPhone(t *testing.T) {
 	}
 
 }
+
+func TestValidSchool(t *testing.T) {
+	user := domains.User{
+		FirstName:  "John",
+		LastName:   "Smith",
+		Email:      "gmail@gmail.com",
+		Phone:      "555-555-0100",
+		IsGuardian: false,
+	}
+	if err := user.Validate(); err != nil {
+		t.Errorf("Check was incorrect, got: %s, expected: nil", err.Error())
+	}
+	//Test valid school
+	user.School = domains.NewNullString("Churchill H.S.")
+	if err := user.Validate(); err != nil {
+		t.Errorf("Check was incorrect, got: %s, expected: nil", err.Error())
+	}
+	user.School = domains.NewNullString("Winston churchill High school")
+	if err := user.Validate(); err != nil {
+		t.Errorf("Check was incorrect, got: %s, expected: nil", err.Error())
+	}
```

Comment:
This is a redundant test to the Montgomery Blair one below so you can remove it (they test basically the same thing).